tests/saitests/sai_qos_tests.py

```diff
@@ -1617,50 +1707,46 @@ def runTest(self):
             pkt_dst_mac2 = def_vlan_mac
             pkt_dst_mac3 = def_vlan_mac
 
-        try:
-            pkt = construct_ip_pkt(packet_length,
-                                pkt_dst_mac,
-                                src_port_mac,
-                                src_port_ip,
-                                dst_port_ip,
-                                dscp,
-                                src_port_vlan,
-                                ecn=ecn,
-                                ttl=ttl)
-            dst_port_id = self.get_rx_port(
-                src_port_id, pkt_dst_mac, dst_port_ip, src_port_ip, dst_port_id, src_port_vlan
-            )
-
-            # create packet
-            pkt2 = construct_ip_pkt(packet_length,
-                                    pkt_dst_mac2,
-                                    src_port_mac,
-                                    src_port_ip,
-                                    dst_port_2_ip,
-                                    dscp,
-                                    src_port_vlan,
-                                    ecn=ecn,
-                                    ttl=ttl)
-            dst_port_2_id = self.get_rx_port(
-                src_port_id, pkt_dst_mac2, dst_port_2_ip, src_port_ip, dst_port_2_id, src_port_vlan
-            )
+        pkt = get_multiple_flows(
+              self,
+	      pkt_dst_mac,
```

Comment:
there are many tabs in fix, better to replace with space.

Reply:
Done, replaced all tabs with 8 spaces.